module-system/sov-modules-macros/src/dispatch/derive_rpc.rs

```diff
@@ -149,7 +149,7 @@ impl RpcImplBlock {
                     .filter(|arg| arg.to_string() != quote! { self }.to_string());
                 quote! {
                     #signature {
-                        <#type_name <#(#generics_params)*,> as ::sov_modules_api::ModuleInfo>::new().#method_name(#(#arg_values),*)
+                        <#type_name <#(#generics_params)*,> as std::default::Default>::default().#method_name(#(#arg_values),*)
```

Comment:
Shouldn't it be `::std::default::Default` so it won't collide if someone has module `std` ?

@bkolad I am not sure about this one.